### ISSUE TITLE
Need to adjust SAOE emissions factors

### ISSUE THREAD

**Issue by fvitt** [ESCOMP/CAM#860]
### Issue Type

Other (please describe below)

### Issue Description

After the bug fix for emissions in tag cam6_3_119 the emissions factors for SOAE, used in configurations without full chemistry, need to be adjusted.

The emissions bug is described in issue #538 
 

### Will this change answers?

Yes

### Will you be implementing this yourself?

Yes

**Comment by fvitt** [comment]
From @Duseong :

 I did a couple of simulations again to reduce the SOA burden in CAM with the new SOA scheme. In the plot below, the red line is SOA after the bug fix, and the purple dotted line is SOA after the bug fix with new SOA yields, you can see our original run (green dash-dot) and the new bug fix run with new factors are very similar.

![unnamed](https://github.com/ESCOMP/CAM/assets/9341606/042bee4a-aa1a-4fd6-8f17-a09054207e16)

I also checked global mean radiation fields, they are again very similar, as SOA burdens are very similar after re-adjusting the SOA burden.

New recommended namelist setting are:
```
srf_emis_specifier             =
         'SOAE -> 2.5592*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_BENZENE_anthro_surface_1750-2015_0.9x1.25_c20170608.nc',
         'SOAE -> 2.5592*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_BENZENE_bb_surface_1750-2015_0.9x1.25_c20170322.nc',
         'SOAE -> 0.5954*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_ISOP_bb_surface_1750-2015_0.9x1.25_c20170322.nc',
         'SOAE -> 5.1004*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_MTERP_bb_surface_1750-2015_0.9x1.25_c20170322.nc',
         'SOAE -> 8.2367*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_TOLUENE_anthro_surface_1750-2015_0.9x1.25_c20170608.nc',
         'SOAE -> 8.2367*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_TOLUENE_bb_surface_1750-2015_0.9x1.25_c20170322.nc',
         'SOAE -> 6.5013*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_XYLENES_anthro_surface_1750-2015_0.9x1.25_c20170608.nc',
         'SOAE -> 6.5013*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_XYLENES_bb_surface_1750-2015_0.9x1.25_c20170322.nc',
         'SOAE -> 8.5371*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_IVOC_anthro_surface_1750-2015_0.9x1.25_c20170608.nc',
         'SOAE -> 8.5371*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_IVOC_bb_surface_1750-2015_0.9x1.25_c20170322.nc',
         'SOAE -> 16.6500*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_SVOC_anthro_surface_1750-2015_0.9x1.25_c20170608.nc',
         'SOAE -> 16.6500*/glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/CMIP6_emissions_1750_2015/emissions-cmip6_SVOC_bb_surface_1750-2015_0.9x1.25_c20170322.nc'



  megan_specifier = 'SOAE = 0.5954*isoprene + 5.1004*(carene_3 + pinene_a + thujene_a + bornene +',
      ' terpineol_4 + terpineol_a + terpinyl_ACT_a + myrtenal + sabinene + pinene_b + camphene +',
      ' fenchene_a + limonene + phellandrene_a + terpinene_a + terpinene_g + terpinolene +',
      ' phellandrene_b + linalool + ionone_b + geranyl_acetone + neryl_acetone + jasmone +',
      ' verbenene + ipsenol + myrcene + ocimene_t_b + ocimene_al + ocimene_c_b + 2met_nonatriene) + ',
      ' 12.3942*(farnescene_a + caryophyllene_b + acoradiene + aromadendrene + bergamotene_a +',
      ' bergamotene_b + bisabolene_a + bisabolene_b + bourbonene_b + cadinene_d + cadinene_g +',
      ' cedrene_a + copaene_a + cubebene_a + cubebene_b + elemene_b + farnescene_b +',
      ' germacrene_B + germacrene_D + gurjunene_b + humulene_a + humulene_g + isolongifolene +',
      ' longifolene + longipinene + muurolene_a + muurolene_g + selinene_b + selinene_d +',
      ' nerolidol_c + nerolidol_t)'

```

**Comment by adamrher** [comment]
I didn't expect that https://github.com/ESCOMP/CAM/issues/538 changed the climate, but now seeing this issue, it seems it did? Can someone explain how? @Duseong do you have some runs we can run the ADF diagnostics on?

**Comment by Duseong** [comment]
@adamrher Just for a quick look, I made a table below (one-year run for 2013, nudged). 
After bug fix #538, SOA burden increased significantly, so we came up with the new factors above. 
It looks like the changes are small when it comes to radiation fields, but I still think it should be corrected to avoid any potential problems. 


<html>
<body>
<google-sheets-html-origin><style type="text/css"></style>

Field | CAM-chem | CESM2.2 default SOA | NEW SOA (CAM6_3_093) | NEW SOA (CAM6_3_119) | NEW SOA (This issue)
-- | -- | -- | -- | -- | --
Burden (SOA) | 1022 | 948 | 1027 | 1277 | 1022
Burden (SOG) | 484 | 118 | 129 | 141 | 132
Burden (BC) | 117 | 131 | 111 | 109 | 109
Burden (POM) | 587 | 704 | 574 | 559 | 560
FSNT | 236.7 | 237.7 | 237.3 | 237.3 | 237.3
FLNT | 238.7 | 239.2 | 239.3 | 239.3 | 239.3
FSNTOA | 238.6 | 239.7 | 239.3 | 239.3 | 239.3
SWCF | -50.5 | -49.6 | -49.8 | -49.8 | -49.9
LWCF | 22.2 | 21.7 | 21.6 | 21.6 | 21.7


</body>
</html>

**Comment by adamrher** [comment]
I'm surprised how little SWCF is changing across these simulations. I wouldn't think that nudging would mitigate the sensitivity to SOA, but it's possible it does. I'd like to see free running AMIP runs for cam6_3_118, cam6_3_119 and cam6_3_119+new SAO tunings. I'm not asking you to do this, I'm just putting this out there in the void.
